src/components/__tests__/IssueQueue.spec.ts

```diff
@@ -296,4 +298,20 @@ describe('issueQueue', () => {
     const filterOptionEL = defaultFilterEL.find('span.badge');
     expect(filterOptionEL.text()).toBe('Affected Component : test');
   });
+
+  it('should render create_dt in UTC format', async () => {
+    Settings.defaultZone = 'Europe/Madrid';
+    const wrapper = mountWithConfig(IssueQueue, {
+      props: {
+        issues: [{ ...mockData[0], created_dt: '2021-07-29T22:50:50Z' }],
+        isLoading: false,
+        isFinalPageFetched: false,
+        total: 10,
+      },
+    });
+
+    const dateEl = wrapper.findAll('td')[2];
+    expect(dateEl.exists()).toBeTruthy();
+    expect(dateEl.text()).toBe('2021-07-29');
```

Comment:
Shouldn't the test actually confirm the time zone is UTC?

Reply:
And is doing it, I'm setting the timezone to `Madrid` https://github.com/RedHatProductSecurity/osim/blob/052468b74d0e9f2e3705461c185b58d0e9b5b73c/src/components/__tests__/IssueQueue.spec.ts#L303
That means that UTC time `2021-07-29T22:50:50Z` would become `2021-07-39T00:50:50Z` on GMT+2.

So expecting the date to be `2021-07-29` means that is **still UTC**.

You can test it yourself by removing the line 303 and seeing the test fail. Do you have any suggestion in other way of testing it? Since the component only shows date and no time its the only one I thought of

Reply:
Oh I didn't see that line with the zone.
Looks good.